src/microbe_stage/components/MicrobeAI.cs

```diff
@@ -26,6 +26,8 @@ public struct MicrobeAI
 
     public float PursuitThreshold;
 
+    public float Strain;
```

Comment:
At least I think this is like wrong, the data should be read from the component it is in and not duplicated.